dicom_utils/types.py

```diff
@@ -515,6 +516,42 @@ def get_required_tags() -> List[Tag]:
         return [*Laterality.get_required_tags(), *ViewPosition.get_required_tags()]
 
 
+@dataclass(frozen=True)
+class PixelSpacing:
+    r"""Represents detector pixel spacing in mm."""
+    row: float
+    col: float
+
+    @classmethod
+    def from_str(cls, string: str) -> "PixelSpacing":
+        # value will be of from [row, col] in mm
+        values = string.strip("[]").split(", ")
```

Comment:
Maybe it'd be safer to just split on `,` (no space) since spaces are ignored by `float(x)` anyways? Elsewhere I had written some code to auto-identify the delimiter (i.e. in case of `|`, `;`, etc.). I don't remember if I actually ran across different kinds of delimiters or if I was just being paranoid.

Reply:
We could also use a regex to ensure a more reliable match - something like `([\d.]+)\D*([\d.]+)`

Reply:
That's not a bad idea. I think we'd want `([\d.]+)\D+([\d.]+)` (i.e. `D+` instead of `D*`) since at least one non-digit should be found?